src/main/java/org/openpnp/gui/tablemodel/PlacementsTableModel.java

```diff
@@ -117,6 +117,16 @@ else if (columnIndex == 5) {
             else if (columnIndex == 6) {
                 placement.setType((Type) aValue);
             }
+            else if (columnIndex == 8) {
+                if (aValue.toString().compareTo("Yes")==0)
```

Comment:
We typically render boolean options as a checkbox by making the column class Boolean. Is there a reason you used these text strings?